### ISSUE TITLE
ClassNotFoundException in ionic project

### ISSUE THREAD

**Issue by pranavm27** [comrat/CordovaVideoPlayerActivity#1]
hi

I tried to use this plugin in ionic project 
its giving class not found an exception when I try to play an mp4 video.

![image](https://user-images.githubusercontent.com/6630080/35085176-736062c8-fc4d-11e7-8d3a-573220270b48.png)

`declare let CordovaVideoPlayerActivity: any;
`
```
CordovaVideoPlayerActivity.play(
    "file:///android_asset/www/movie.mp4",
    function () {
        console.log("video completed");
    },
    function (err) {
        console.log(err);
    }
```

**Comment by comrat** [comment]
Hello!
Actually I haven't try ionic before. Unfortunately I've not much time now and can check it later.
If you have any suggestions please let me know

**Comment by comrat** [comment]
@pranavm27 finally I've tried to build ionic project by myself and I've reproduce this issue.
Now I've fix it [da6776d1aa1b3aa4bbc705c1bd81f3ab9d2a51d8](https://github.com/comrat/CordovaVideoPlayerActivity/commit/da6776d1aa1b3aa4bbc705c1bd81f3ab9d2a51d8).
Pull master and check it out
